src/lib.rs

```diff
@@ -18,6 +18,10 @@ pub enum Instr {
     Div(Addr, Addr, Addr),
     /// a = b % c
     Rem(Addr, Addr, Addr),
+    /// Jumps program execution by n instructions
+    Jump(i8),
```

Comment:
We could use a larger value (`i16`?) for our unconditional jump, since it wouldn't make the instruction size larger.